kate/src/com.rs

```diff
@@ -673,7 +657,7 @@ mod tests {
 	fn random_cells(cols: usize, rows: usize, percents: usize) -> Vec<Cell> {
 		assert!(percents > 0 && percents <= 100);
 
-		let rng = &mut rand::thread_rng();
+		let rng = &mut ChaChaRng::from_seed([0u8; 32]);
```

Comment:
Wouldn't this defy purpose of random testing?

Reply:
New ticket for this [here](https://linear.app/matic/issue/AVA-151/double-check-rng-options-for-testing-and-rpc)